lib/engine/game/base.rb

```diff
@@ -30,6 +30,46 @@
 
 module Engine
   module Game
+    def self.load(source, at_action: nil, **kwargs)
+      hash_game = nil
+      db_game = nil
+      case source
```

Comment:
maybe game_data = case source
when Hash, String

{ }...
else
{] ...
end

Reply:
Done in 9d89a026f6e0afd0eed322b20ea3cdf3dcfc5531